doc/source/user_guide/computation.rst

```diff
@@ -597,6 +597,18 @@ You can view other examples of ``BaseIndexer`` subclasses `here <https://github.
 
 .. versionadded:: 1.1
 
+One subclass of note within those examples is the ``NonFixedVariableWindowIndexer`` that allows
+rolling operations over a non-fixed offset like a ``BusinessDay``.
+
+.. ipython:: python
+
+   from pandas.core.window.indexers import NonFixedVariableWindowIndexer
```

Comment:
are we adding this to pandas.api.indexers? (i think ok to do this)

Reply:
Sure I can go ahead and add it to `pandas.api.indexers`. I think it's worth exposing